### PR TITLE
Renamed ton units to vmshell units

### PR DESCRIPTION
**Pull request by Oxydixi**

### CODE REVIEW

File: tvm_cli/README.md [hunk 1]
```diff
@@ -334,7 +334,7 @@ List of available options:
 --access_key <ACCESS_KEY>                     Project secret or JWT in Evercloud (dashboard.evercloud.dev).
 --addr <ADDR>                                 Contract address.
 --async_call <ASYNC_CALL>                     Disables wait for transaction to appear in the network after call command.
---balance_in_tons <BALANCE_IN_TONS>           Print balance for account command in tons. If false balance is printed in nanotons.
+--balance_in_tons <BALANCE_IN_VMSHELLS>           Print balance for account command in VMSHELLs. If false balance is printed in VMSHELL.
```
Comment (SilkovAlexander) [inline tvm_cli/README.md:337]: It's better to rename flag to `balance_in_vmshells`

Comment (Oxydixi) [reply tvm_cli/README.md]: Ready

File: tvm_cli/README.md [hunk 2]
```diff
@@ -334,7 +334,7 @@ List of available options:
 --access_key <ACCESS_KEY>                     Project secret or JWT in Evercloud (dashboard.evercloud.dev).
 --addr <ADDR>                                 Contract address.
 --async_call <ASYNC_CALL>                     Disables wait for transaction to appear in the network after call command.
---balance_in_tons <BALANCE_IN_TONS>           Print balance for account command in tons. If false balance is printed in nanotons.
+--balance_in_vmshells <BALANCE_IN_VMSHELLS>           Print balance for account command in VMSHELLs. If false balance is printed in VMSHELL.
```
Comment (SilkovAlexander) [inline tvm_cli/README.md:337]: `If false balance is printed in nanoVMSHELL.`

File: tvm_cli/README.md [hunk 3]
```diff
@@ -1823,7 +1823,7 @@ List of owners must be specified by their public keys in hex format, split by th
 
 `--confirms <confirms_cnt>` - option that sets required number of confirmations. If not specified, is set to 1.
 
-`--local <local_giver_value>` - value that should be transferred from the local giver if wallet is deployed onto the Node SE (in nanotons).
+`--local <local_giver_value>` - value that should be transferred from the local giver if wallet is deployed onto the Node SE (in VMSHELLs).
```
Comment (SilkovAlexander) [inline tvm_cli/README.md:1826]: nanovmshells

File: tvm_cli/README.md [hunk 4]
```diff
@@ -2714,7 +2714,7 @@ This commands allow user to learn how much funds smart contract can consume.
 
 #### 9.8.1. Call fee command
 
-This command executes smart contract call locally, calculates fees and prints table of all fees in nanotons.
+This command executes smart contract call locally, calculates fees and prints table of all fees in VMSHELLs.
```
Comment (SilkovAlexander) [inline tvm_cli/README.md:2717]: nano

File: tvm_cli/README.md [hunk 5]
```diff
@@ -2751,7 +2751,7 @@ Not set rand_seed_block
 
 #### 9.8.2. Deploy fee command
 
-This command executes smart contract deploy locally, calculates fees and prints table of all fees in nanotons.
+This command executes smart contract deploy locally, calculates fees and prints table of all fees in VMSHELLs.
```
Comment (SilkovAlexander) [inline tvm_cli/README.md:2754]: nano

File: tvm_cli/tests/samples/wallet.sol [hunk 6]
```diff
@@ -25,11 +25,11 @@ contract Wallet {
 
     /// @dev Allows to transfer grams to destination account.
     /// @param dest Transfer target address.
-    /// @param value Nanograms value to transfer.
+    /// @param value VMSHELLs value to transfer.
```
Comment (SilkovAlexander) [inline tvm_cli/tests/samples/wallet.sol:28]: nano

**Review by SilkovAlexander — APPROVED** [review]

**Pull request merged**